keyboards/planck/keymaps/callum/rules.mk

```diff
@@ -3,7 +3,7 @@
 #   the appropriate keymap folder that will get included automatically
```

Comment:
Just while you're here, this entire file can be reduced to:
```make
BOOTMAGIC_ENABLE = no
MOUSEKEY_ENABLE = no
CONSOLE_ENABLE = no
COMMAND_ENABLE = yes
MIDI_ENABLE = no
AUDIO_ENABLE = yes
RGBLIGHT_ENABLE = no
```

Reply:
good to know, thanks!